ext/reflection/php_reflection.c

```diff
@@ -2996,7 +2996,7 @@ ZEND_METHOD(reflection_type, isBuiltin)
 	}
 	GET_REFLECTION_OBJECT_PTR(param);
 
-	RETVAL_BOOL(param->arg_info->type_hint != IS_OBJECT);
```

Comment:
where is *intern used in the implemenation of the function? I don't see it. Is it implicit elsewhere? @brzuchal @nikic